Winapp3/Winapp3.ini

```diff
@@ -659,85 +660,114 @@ ExcludeKey22=FILE|%SystemDrive%\Users\All Users\Microsoft\Network\Downloader\|ed
-ExcludeKey79=PATH|%WinDir%\System32\config\|BBI.LOG*
-ExcludeKey80=PATH|%WinDir%\System32\config\|BCD-Template.LOG*
-ExcludeKey81=PATH|%WinDir%\System32\config\|COMPONENTS.LOG*
-ExcludeKey82=PATH|%WinDir%\System32\config\|DEFAULT.LOG*
-ExcludeKey83=PATH|%WinDir%\System32\config\|DRIVERS.LOG*
-ExcludeKey84=PATH|%WinDir%\System32\config\|ELAM.LOG*
-ExcludeKey85=PATH|%WinDir%\System32\config\|SAM.LOG*
-ExcludeKey86=PATH|%WinDir%\System32\config\|SECURITY.LOG*
-ExcludeKey87=PATH|%WinDir%\System32\config\|SOFTWARE.LOG*
-ExcludeKey88=PATH|%WinDir%\System32\config\|SYSTEM.LOG*
-ExcludeKey89=PATH|%WinDir%\System32\config\|userdiff.LOG*
-ExcludeKey90=PATH|%WinDir%\System32\config\bbimigrate\|*.LOG*
-ExcludeKey91=PATH|%WinDir%\System32\config\RegBack\|*.LOG*
-ExcludeKey92=PATH|%WinDir%\System32\config\systemprofile\|NTUSER.*LOG*
-ExcludeKey93=PATH|%WinDir%\System32\config\systemprofile\AppData\Local\DataSharing\Storage\|*.chk;*.log
-ExcludeKey94=PATH|%WinDir%\System32\config\systemprofile\AppData\Local\TileDataLayer\Database\|EDB*.*
-ExcludeKey95=PATH|%WinDir%\System32\SMI\Store\Machine\|SCHEMA.DAT.LOG*
-ExcludeKey96=PATH|%WinDir%\SysWOW64\config\|DEFAULT.*LOG*
-ExcludeKey97=PATH|%WinDir%\SysWOW64\config\|SAM.*LOG*
-ExcludeKey98=PATH|%WinDir%\SysWOW64\config\|SECURITY.*LOG*
-ExcludeKey99=PATH|%WinDir%\SysWOW64\config\|SOFTWARE.*LOG*
-ExcludeKey100=PATH|%WinDir%\SysWOW64\config\systemprofile\|NTUSER.*LOG*
-ExcludeKey101=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\|*.tmp
-ExcludeKey102=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\DataSharing\Storage\|*.chk;*.log;*.jrs
-ExcludeKey103=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\Microsoft\Windows\WebCache.old\|*.chk;*.jrs
-ExcludeKey104=PATH|%WinDir%\SysWOW64\config\systemprofile\AppData\Local\Microsoft\Windows\WebCache\|*.chk
+ExcludeKey26=FILE|%WinDir%\|PFRO.log
+ExcludeKey27=FILE|%WinDir%\|WindowsUpdate.log
+ExcludeKey28=FILE|%WinDir%\Performance\WinSAT\|winsat.log
+ExcludeKey29=FILE|%WinDir%\security\database\|edb.chk
+ExcludeKey30=FILE|%WinDir%\ServiceProfiles\NetworkService\AppData\Local\Temp\|MpCmdRun.log
+ExcludeKey31=FILE|%WinDir%\System32\|Local_LLU.log
+ExcludeKey32=FILE|%WinDir%\System32\|LocalGroupAdminAdd.log
+ExcludeKey33=FILE|%WinDir%\System32\|Network_LLU.log
+ExcludeKey34=FILE|%WinDir%\System32\Msdtc\|MSDTC.LOG
+ExcludeKey35=FILE|%WinDir%\System32\Msdtc\Trace\|dtctrace.log
+ExcludeKey36=FILE|%WinDir%\System32\sru\|SRU.chk
+ExcludeKey37=FILE|%WinDir%\System32\sysprep\Panther\|setupact.log
+ExcludeKey38=FILE|%WinDir%\System32\sysprep\Panther\|setuperr.log
+ExcludeKey39=FILE|%WinDir%\System32\sysprep\Panther\IE\|setupact.log
+ExcludeKey40=FILE|%WinDir%\System32\sysprep\Panther\IE\|setuperr.log
+ExcludeKey41=PATH|%AppData%\steelseries-engine-3-client\|*.log
+ExcludeKey42=PATH|%CommonAppData%\Microsoft\Provisioning\|Microsoft-Desktop-Provisioning-Sequence.dat.LOG*
+ExcludeKey43=PATH|%CommonAppData%\Microsoft\Search\Data\Applications\Windows\|*.chk

```

Comment:
Good to know ExcludeKeys ignore semicolons. Will remember this for the future.

Reply:
Would be great, if there would be an page, where we document the limitations of CCleaner.
However, I don't know the behavoir of newer versions.